phoebe/backend/universe.py

```diff
@@ -2432,7 +2435,7 @@ def is_convex(self):
     def needs_recompute_instantaneous(self):
         # recompute instantaneous for asynchronous spots, even if meshing
         # doesn't need to be recomputed
-        return self.needs_remesh or (not self.is_single and len(self.features) and self.F != 1)
+        return self.needs_remesh or (len(self.features) and self.F != 1.0) or (len(self.features) and self.is_single)
```

Comment:
This just reads a little more intuitively to me 🤷‍♂️   The comment above probably should be updated as well to explain why the single case needs recomputing.
```suggestion
        return self.needs_remesh or (len(self.features) and (self.is_single or self.F != 1.0))
```